client/tx/tx.go

```diff
@@ -9,18 +9,19 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
-	"github.com/cosmos/cosmos-sdk/client"
-	"github.com/cosmos/cosmos-sdk/examples/basecoin/app" // XXX: not good
+	"github.com/cosmos/cosmos-sdk/client" // XXX: not good
```

Comment:
why not ?

Reply:
again this is antiquated code this has been removed - there is weird thing (error?) in go-imports that will just shift your comment when you save the dep with a comment was beside has been removed - the XXX was in reference to the client referencing the basecoin examples input